crates/turbopack-css/src/chunk/writer.rs

```diff
@@ -36,6 +36,18 @@ pub async fn expand_imports(
                     close,
                 ));
             }
+            Some(CssImport::Composes(composed_chunk_item)) => {
```

Comment:
There should be some kind of deduplication of imports here (and in CssImport::Internal) to avoid including a CSS chunk item multiple times into a chunk.